.github/CODEOWNERS

```diff
@@ -9,8 +26,11 @@
 algorithms/           @Qiskit/terra-core @manoelmarques @woodsp-ibm
```

Comment:
Would it make sense also add `test/python/<folder>` to the the rules since the PRs usually touch the accordings tests, too?

Reply:
These already do that, I believe - the matches are partial, so it matches `algorithms/` anywhere in the tree.